tests/test_buku.py

```diff
@@ -924,8 +930,8 @@ def test_convert_bookmark_set(export_type, exp_res, monkeypatch):
 
     bms = [
         (1, "http://example.com", "", ",", "", 0),
-        (1, "http://example.org", None, ",", "", 0),
-        (2, "http://google.com", "Google", ",", "", 0),
+        (2, "http://example.org", None, ",bar,baz,foo,", "", 0),
+        (3, "http://google.com", "Google", ",bar,baz,foo,", "", 0),
```

Comment:
Adding tags to test their export as well. (…Also fixing indices since these wouldn't be encountered in actual data)